backend/unitTests/getUserItemsTest.py

```
@@ -0,0 +1,299 @@
+    
+    # Since FastAPI validates path params as int, we need to test the actual vulnerability
+    # The vulnerability is in the backend SQL query, not the path validation
+    print("   Note: FastAPI path validation blocks non-integer user_id values")
+    print("   Testing with integer values that could exploit the SQL injection...")
+    
+    # Test with values that would work if SQL injection was possible
+    test_values = [1, 999, -1]
+    
+    for test_val in test_values:
+        try:
+            response = requests.get(f"{BASE_URL}/api/items/{test_val}")
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)